Description: Adds missing files in MANIFEST.in
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no

Index: ceilometer/MANIFEST.in
===================================================================
--- ceilometer.orig/MANIFEST.in	2012-11-27 07:11:10.508578320 +0000
+++ ceilometer/MANIFEST.in	2012-11-28 12:25:22.000000000 +0000
@@ -4,3 +4,17 @@
 exclude .gitreview
 
 global-exclude *.pyc
+include ceilometer/api/static/LICENSE.d3.v2.min.js
+include ceilometer/api/static/LICENSE.rickshaw
+include ceilometer/api/static/bootstrap/css/bootstrap-responsive.css
+include ceilometer/api/static/bootstrap/css/bootstrap.css
+include ceilometer/api/static/bootstrap/img/glyphicons-halflings-white.png
+include ceilometer/api/static/bootstrap/img/glyphicons-halflings.png
+include ceilometer/api/static/bootstrap/js/bootstrap.js
+include ceilometer/api/static/d3.v2.js
+include ceilometer/api/static/jquery-1.8.3.js
+include ceilometer/api/static/rickshaw.css
+include ceilometer/api/static/rickshaw.js
+include ceilometer/api/templates/list_event.html
+include ceilometer/storage/sqlalchemy/migrate_repo/README
+include ceilometer/storage/sqlalchemy/migrate_repo/migrate.cfg
