Description: Uses mongodb by default in the connection directive
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-04-03

--- ceilometer-2014.1~rc1.orig/etc/ceilometer/ceilometer.conf.sample
+++ ceilometer-2014.1~rc1/etc/ceilometer/ceilometer.conf.sample
@@ -631,7 +631,7 @@
 # Deprecated group/name - [DEFAULT]/sql_connection
 # Deprecated group/name - [DATABASE]/sql_connection
 # Deprecated group/name - [sql]/connection
-#connection=sqlite:////ceilometer/openstack/common/db/$sqlite_db
+connection=mongodb://localhost:27017/ceilometer
 
 # The SQLAlchemy connection string used to connect to the
 # slave database (string value)
